packages/i18n/data/en.json

```diff
@@ -156,7 +156,7 @@
   "QuickAccess.useProject": "Switch to project \"{projectName}\"",
   "TopNavigation.labelLoading": "Processing...",
   "UserSettingsMenu.logout": "Logout",
-  "UserSettingsMenu.manageOrganizations": "Manage Organizations",
+  "UserSettingsMenu.manageOrganizations": "",
```

Comment:
how come this is empty?

Reply:
Because you the one in core is the correct one 👍